spec/controllers/provider_management/external_users_controller_spec.rb

```diff
@@ -206,38 +203,34 @@ def post_to_create_external_user_action(options={})
   end
 
   describe "PUT #update_password" do
+    let(:password) { 'password123' }
+    let(:password_confirm) { password }
+    subject { put :update_password, params: { provider_id: provider, id: external_user, external_user: { user_attributes: { password: password, password_confirmation: password_confirm } } } }
 
-    context 'when valid' do
-
-      before(:each) do
-        put :update_password, params: { provider_id: provider, id: external_user, external_user: { user_attributes: { password: 'password123', password_confirmation: 'password123' } } }
-        external_user.reload
-      end
+    before(:each) { travel(-6.months) { external_user } }
```

Comment:
It probably does not matter but to emulate frozen time that was used for external user creation prior to this you use `travel_to` (`travel` does not freeze time [from what i am reading](https://api.rubyonrails.org/v5.2.4/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel))

Reply:
The test was checking that the updated time is not exactly the same as the frozen time. There was a problem with this as the milliseconds part of the time is lost by `travel_to`.

I have changed it just to see if it has changed (or not changed as in the test further down).